msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.props

```diff
@@ -57,6 +57,10 @@ Copyright (C) 2013-2014 Xamarin. All rights reserved.
 		<IsAppExtension Condition="'$(IsAppExtension)' == ''">False</IsAppExtension>
 	</PropertyGroup>
 
+	<PropertyGroup>
+		<MtouchTargetsEnabled>true</MtouchTargetsEnabled>
```

Comment:
Shouldn't this only set the property value to true if it's not already set? Otherwise how does VS or w/e override it?